### ISSUE TITLE
[Bug]Kodi 18.3 breaks openinfo

### ISSUE THREAD

**Issue by JayDee696969** [a4k-openproject/script.extendedinfo#12]
**Describe the bug**
Ok, so it seems Kodi 18.3 breaks openinfo. 

**To Reproduce**
Run Openinfo, select play to bring up the players list, choose a player and Kodi crashes.
Reverting to Kodi 18.2 and try again and works fine.

```
2019-06-30 09:49:08.647 T:14504   ERROR: Exception in thread Thread-3:
                                            Traceback (most recent call last):
                                              File "C:\Program Files\Kodi\system\python\Lib\threading.py", line 801, in __bootstrap_inner
                                                self.run()
                                              File "C:\Program Files\Kodi\system\python\Lib\threading.py", line 754, in run
                                                self.__target(*self.__args, **self.__kwargs)
                                              File "C:\Users\Jay\AppData\Roaming\Kodi\addons\script.extendedinfo\resources\lib\DialogBaseInfo.py", line 116, in get_youtube_vids
                                                result = YouTube.search_youtube(search_str, limit=10)
                                              File "C:\Users\Jay\AppData\Roaming\Kodi\addons\script.extendedinfo\resources\lib\YouTube.py", line 78, in search_youtube
                                                videos = handle_youtube_videos(results['items'], extended=extended)
                                            KeyError: 'items'
```

**Comment by lukehansford** [comment]
Can confirm crashing in 18.3 as described.

**Comment by diabl0w** [comment]
Can confirm, same crash log running latest CoreELEC

**Comment by diabl0w** [comment]
right after the lines already reported, I also have the following lines

`
2019-07-14 23:05:33.371 T:3075453808   ERROR: NEWADDON Unknown Video Info Key "fanart"
2019-07-14 23:05:33.371 T:3075453808   ERROR: NEWADDON Unknown Video Info Key "name"
2019-07-14 23:05:33.371 T:3075453808   ERROR: NEWADDON Unknown Video Info Key "poster"
2019-07-14 23:05:33.371 T:3075453808   ERROR: NEWADDON Unknown Video Info Key "tmdb"`

**Comment by drinfernoo** [comment]
Closed in 465deb72e58c8dba7419899382cab4e1dd296ef6.
